flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/UnionTest.scala

```diff
@@ -40,10 +40,9 @@ class UnionTest extends TableTestBase {
          |CREATE TABLE t1 (
          |  id int,
          |  ts bigint,
-         |  name string,
+         |  name varchar(32),
```

Comment:
If you want to delete one of them, I think it's better to keep the one that has the same type with other tables.